facility-manager.html

```diff
@@ -105,10 +106,12 @@
 <h2><a class="anchor" id="permissions"></a>Permissions on OMERO</h2>
 
 <p>
-    There are three levels of user permission in OMERO, Administrator, Group
-    Owner and Member. Each level of permission allows different actions to be
-    performed on a user's own, or other user's data, and certain
-    administrative tasks.
+    There are three fixed levels of user permissions in OMERO, Administrator, Group
+    Owner and Member as well as one adjustable level, Administrator with restricted
+    privileges (Restricted Administrators). Each level of permissions allows different
+    actions to be performed on a user's own, or another user's data, and certain
+    administrative tasks. Permission levels of Restricted Administrators
+    are determined at the point of their creation by the creating Administrator.
```

Comment:
can we call the creating Administrator as the system administrator ?

Reply:
I'm not sure 'system administrator' means anything different than 'Administrator'. But to clarify, we could say: "Restricted Administrators can be created and edited by Administrators or other Restricted Administrators.This includes setting of their permission levels."